recOrder/scripts/cli.py

```diff
@@ -27,36 +29,31 @@ def cli():
     type=click.Choice(["position", "channel", "p", "c"]),
     help="Layers as 'position' ('p') or 'channel' ('c')",
 )
-def view(filename, position=None, layers=None):
+def view(filename, positions=None, layers=None):
     """View a dataset in napari"""
-    print(f"Reading file:\t {filename}")
+    click.echo(f"Reading file:\t {filename}")
+    print_info(filename)
 
-    # Workaround waveorder #97
-    # Create napari Viewer before other imports
-    v = napari.Viewer()
-    v.close()
-    from waveorder.io import WaveorderReader
-
-    reader = WaveorderReader(filename)
-    print_reader_info(reader)
+    reader = read_micromanager(filename)
```

Comment:
@ziw-liu I think I'm going to need an `if` statement here, so that `recorder view` will work with v0.4 zarrs. 

What would you suggest? Should I use `_infer_format`?

Reply:
Probably yes... `_infer_format()` is 'private' but does exactly what we want here. A more fragile solution is to capture the stdout of `print_reader_info()` that has already called `_infer_format()` and printed the format.

Reply:
Thanks. On my first pass I used `_infer_format`, but please feel free to change this or make a suggestion.